plugins/modules/systemd_info.py

```diff
@@ -275,7 +288,8 @@ def get_category_base_props(category):
         'service': ['FragmentPath', 'UnitFileState', 'UnitFilePreset', 'MainPID', 'ExecMainPID'],
         'target': ['FragmentPath', 'UnitFileState', 'UnitFilePreset'],
         'socket': ['FragmentPath', 'UnitFileState', 'UnitFilePreset'],
-        'mount': ['Where', 'What', 'Options', 'Type']
+        'mount': ['Where', 'What', 'Options', 'Type'],
+        'timer': ['FragmentPath', 'UnitFileState', 'UnitFilePreset']
```

Comment:
We missed that in the `mount` line, so now it has been changed (unnecessarily, we might argue). So, let's do it for the next time ;-)
```suggestion
        'timer': ['FragmentPath', 'UnitFileState', 'UnitFilePreset'],
```